runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -340,7 +340,9 @@ function LanguageTree:_get_injections()
 
         -- Allow for captured nodes to be used
         if type(content) == 'number' then
-          content = { match[content]:range() }
+          for _, node in ipairs(match[content]) do
+            vim.list_extend(ranges, { node:range() })
```

Comment:
Is `list_extend` right here? Can't we have `ranges[#ranges+1] = node:range()`?

Reply:
List extend here is mandatory because `node:range()` retunrs multiple values instead of a table.

Reply:
So then why not `ranges[#ranges+1] = {node:range()}`?

Isn't `ranges` a list of ranges (a.k.a a list of integer lists)? `vim.list_extend` will just make ranges a list of integers?

Reply:
You were indeed right, it should be a insert and not an extend. The previous implementation is just a bit weird because it resulted in `ranges` being a single range, even though it should have been a table of ranges (of length 1). So the previous implementation should have been `{ { match[content]:range() } }`. I think it only worked in this case because of some leniency further down.